docs/source/demos/fraud.ipynb

```diff
@@ -68,7 +68,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X, y = evalml.demos.load_fraud(n_rows=1000)"
+    "X, y = evalml.demos.load_fraud(n_rows=5000)"
```

Comment:
Added `n_rows=5000` as stated in the perf test doc [here](https://alteryx.quip.com/ZDhLAlkjiVR4/Fraud-and-Lead-Scoring-analysis) for performance improvement